store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedStoreTest.java

```diff
@@ -1242,8 +1243,8 @@ public void shouldGetAllElementsFromSelectedRemoteGraphWhileHasConflictingSchema
         final Entity A = getEntityA();
         final Entity B = getEntityB();
 
-        final List<Entity> expectedA = asList(A);
-        final List<Entity> expectedB = asList(B);
+        final List<Entity> expectedA = singletonList(A);
+        final List<Entity> expectedB = singletonList(B);
```

Comment:
While making these sorts of changes, it would also be good to remove use of the Guava import `com.google.common.collect.Sets`.

Reply:
We can look at all of these later maybe.